packages/core/src/app/order/OrderSummary.tsx

```diff
@@ -26,8 +41,71 @@ const OrderSummary: FunctionComponent<OrderSummaryProps & OrderSummarySubtotalsP
     total,
     ...orderSummarySubtotalsProps
 }) => {
+    const [loyaltiPoints, setLoyaltyPoints] = useState<Reward>();
+    const { applyGiftCertificate } = orderSummarySubtotalsProps;
     const nonBundledLineItems = useMemo(() => removeBundledItems(lineItems), [lineItems]);
 
+    const loadLoyaltyPoints = useCallback(() => {
+      const session_data = sessionStorage.getItem("stencil_graphql");
+      let stencil_graphql:Stencil_Graphql;
+
+      if(session_data)
+        stencil_graphql = JSON.parse(session_data);
+      else 
+        return;
+
+      if(stencil_graphql.storefront_api) {
+        const graphql = JSON.stringify({
+          query: `
+            {
+              customer {
+                attributes {
+                  attribute(entityId: 41) {
```

Comment:
oh thanks for pointing this, yeah I forgot to discuss this with Michael as we can't call management APIs from checkout, so have to ask michael if this entity ID is same for rewards attribute or it can be changed.
So yeah I will check with Michael